cirq/google/engine/engine.py

```diff
@@ -347,7 +349,9 @@ def run_batch(
             for a circuit are listed in the order imposed by the associated
             parameter sweep.
         """
-        if not params_list or len(programs) != len(params_list):
+        if params_list is None:
+            params_list = [dict()] * len(programs)
+        if len(programs) != len(params_list):
```

Comment:
The line above ensures `params_list` and `programs` are of equal length, so this `if` can be an `elif`.

Reply:
done